server/controllers/article.js

```diff
@@ -27,9 +29,24 @@ class Article {
         userid: request.userId,
         description,
         readtime,
-        images: images.split(','),
+        images: images ? images.split(',') : [],
       });
-      return goodHttpResponse(response, 201, 'Article Created', newArticle);
+      if (request.body.tags) {
+        const combinedTags = request.body.tags.toLowerCase();
```

Comment:
is tags a string?

Reply:
Yes, it is. They are bundled and sent as a string. I then split the tags into an array